lib/ecto/adapters/sql/sandbox.ex

```diff
@@ -1,329 +1,299 @@
 defmodule Ecto.Adapters.SQL.Sandbox do
```

Comment:
DBConnection was designed for ownership rather than old Sandbox so implementation is not ideal but tests pass - though logging may be tricky here.